src/omotes_orchestrator/main.py

```diff
@@ -206,12 +210,12 @@ def start(self) -> None:
 
         self.celery_if.start()
 
-        self.jobs_broker_if.start()
-        self.jobs_broker_if.add_queue_subscription(
```

Comment:
Move this logic (including the handlers & wrappers) to worker_interface.py; similar setup to sdk_interface.py and celery_interface.py.